README.md

```diff
@@ -29,51 +35,48 @@ WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 GNU General Public License for more details.
 
-You should have received a copy of the GNU General Public License
-along with Octave; see the file COPYING.  If not, see
-<https://www.gnu.org/licenses/>.
+You should have received a copy of the 
+[`GNU General Public License`](https://github.com/gnu-octave/octave/blob/default/COPYING)
```

Comment:
The GuitHub mirror is just that (a mirror of the main repository).
We shouldn't hard-code paths to the mirror repository here.